tools/ci_connector_ops/ci_connector_ops/qa_engine/outputs.py

```diff
@@ -3,13 +3,18 @@
 #
 
 
+from datetime import datetime
+
 import pandas as pd
 
 from .models import ConnectorQAReport
 
-def persist_qa_report(qa_report: pd.DataFrame, path: str, public_fields_only: bool =True):
+
+def persist_qa_report(qa_report: pd.DataFrame, path: str, public_fields_only: bool = True) -> str:
+    report_generation_date = datetime.strftime(qa_report["report_generation_datetime"].max(), "%Y%m%d")
```

Comment:
Is `report_generation_datetime` different for each row in the qa_report?

Reply:
No its the same value on each row for a report.